art/loggers.py

```diff
@@ -1,11 +1,37 @@
 import os
+import sys
+import uuid
+from datetime import datetime
+from pathlib import Path
 from typing import Optional, Union
 
-from neptune import init_run
-from neptune.exceptions import MissingFieldException
 import numpy as np
-from wandb import log, save, Image
 from lightning.pytorch.loggers import NeptuneLogger, WandbLogger
+from loguru import logger
```

Comment:
I'm not sure whether keeping loguru and Neptune/wanb/... is a good idea. I have already confused the name "logger" several times, and it'll be more confusing if we keep it in the same file. Maybe we could name Neptune etc. as experiment trackers, not loggers themselves?

Reply:
My contrargument would be this screen
![image](https://github.com/SebChw/art/assets/66476124/0d6cbecb-1ae1-4aa5-b9ba-9bb4b0f20e86)

We are logging to a tracking system.

Reply:
okay, I won't argue further